docs/Maintainers.md

```diff
@@ -93,3 +93,4 @@ instead, it should be brought up with the Community Maintenance Committee.
 | Shruti Ratnam         | shruti25ratnam      | Attestation                    |
 | Cheng-mean Liu        | soccerGB            | Attestation, SGX               |
 | Bruce Campbell        | yakman2020          | Windows, SGX                   |
+| Brett McLaren         | BRMcLaren           |                                |
```

Comment:
Should the areas say packaging/build/CI instead of being blank?

Reply:
@dthaler Brett may end up working on some other tasks unrelated to packaging/build/CI, so I left it blank.
I'm genuinely not really sure what to put here for new people! What does everyone think? :)

Reply:
If it's blank, it means the person doesn't need to be added as a reviewer on anything.
If it's not blank, it means the person should be added as a review on anything in that area.

Reply:
Sounds good to me. For now I'd say lets leave it blank, since Brett is just now onboarding.  Thanks so much, Dave!